dependencies.gradle

```diff
@@ -43,6 +43,7 @@ ext {
     autodispose = "com.uber.autodispose:autodispose:$autodisposeVersion"
 
     archComopnentsLifecycle = "android.arch.lifecycle:runtime:$archComponentsVersion"
+    archComopnentsExtensions = "android.arch.lifecycle:extensions:$archComponentsVersion"
```

Comment:
Comopnents -> Components

Reply:
thanks @jshvarts I fixed the typo and removed the runtime import, as the extensions already depend on it so it's not required to explicitly add it